pyvo/dal/adhoc.py

```diff
@@ -629,6 +651,13 @@ def get_first_proc(self):
             return proc
         raise IndexError("No processing service found in datalink result")
 
+    @classmethod
+    def from_result_url(cls, result_url, *, session=None, original_row=None):
+        res = super(DatalinkResults, cls).from_result_url(
+            result_url, session=session)
```

Comment:
can be written in one line, maybe more readable that way
```suggestion
        res = super().from_result_url(result_url, session=session)
```